cmd/truss/main.go

```diff
@@ -39,7 +64,7 @@ func init() {
 
 	if len(flag.Args()) == 0 {
 		fmt.Fprintf(os.Stderr, "%s: missing .proto file(s)\n", filepath.Base(os.Args[0]))
-		fmt.Fprintf(os.Stderr, "Try '%s --help' for more information.\n", filepath.Base(os.Args[0]))
+		flag.Usage()
```

Comment:
The user has done something wrong. My thought is, why make them issue another command to figure out what they need to do? I wasn't aware that you had put conscious thought into this to make it match other tools, but it's also notable that a tool like `cp` has many more options. And that the example you cited is GNU specific:

#### BSD on OS X
```
➜  go-truss git:(add_version_tag) ✗ cp
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
```

I'm not going to get too hung up on it one way or the other, though. I like to try to be a steward for the user and not make them do things unnecessarily when I can save them some work, but the unadorned output is not the prettiest, I'll admit.

Since we're talking about passing no options to truss, this isn't related, but I think it'd be nice for Truss to scan the current directory for `*.proto` files and attempt to process them. Since Truss is so directory-oriented as it is, and has opinions on where the file must live (i.e., in the GOPATH), it seems a natural extension.